pkg/kyverno/validate/command.go

```diff
@@ -0,0 +1,74 @@
+package validate
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+
+	"github.com/golang/glog"
+
+	policyvalidate "github.com/nirmata/kyverno/pkg/policy"
+
+	v1 "github.com/nirmata/kyverno/pkg/api/kyverno/v1"
+	"github.com/spf13/cobra"
+	"k8s.io/apimachinery/pkg/util/yaml"
+)
+
+func Command() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "validate",
+		Short:   "Validates kyverno policies",
+		Example: "kyverno validate /path/to/policy1 /path/to/policy2",
```

Comment:
Would it be helpful to validate policies under the same folder, i.e., `kyverno validate /path/to/policies`?

Reply:
I can update logic to support both, folder and files - its your call

Reply:
I think it'd be useful, we can validate all best practices with this command :)

Reply:
will add it in

Reply:
Requested changes have been made